adapter/README.md

```diff
@@ -21,45 +21,6 @@ Planned formats:
 Note that you need to delete existing docker images from your local docker daemon to have recent changes integrated. 
 * For integration testing run `docker-compose -f ../docker-compose.yml -f ../docker-compose.ci.yml up adapter-service adapter-service-it`
 
-## API
```

Comment:
Removed it; I think activating a swagger UI in combination with a hosted version is far more valuable.
Not sure what you think about it @mathiaszinnen , I can add it again if you think it makes sense

Reply:
In the long run, I would agree. Until then, I would rather keep it. Especially since I kept it up-to-date until now.